vite.config.ts

```diff
@@ -16,7 +16,7 @@ export default defineConfig(({ mode }) => {
 
   const PISA_URL = env.PISA_URL
   const enablePisa = !!PISA_URL
-  if (enablePisa) console.log('Pisa included in the build.')
+  if (enablePisa) console.log('Pisa is enabled.')
```

Comment:
* [ ] FYI I realized that in practice this message is noisy. Personally I'd like to get rid of it.

Reply:
Let's see, if I remove it eventually.